frameworks/kafka/universe/marathon.json.mustache

```diff
@@ -59,6 +59,8 @@
     "CONFIG_TEMPLATE_PATH": "kafka-scheduler",
     "KAFKA_VERSION_PATH": "kafka_2.11-0.10.2.1",
 
+    "KAFKA_ZOOKEEPER_URI": "{{kafka.kafka_zookeeper_uri}}",
```

Comment:
It's being passed to the scheduler, so that the scheduler can fill in a reasonable value if it's empty/unset. The scheduler then passes the result to the tasks. See code in KafkaService.java above.